### ISSUE TITLE
Missing fields in the json response for many sources

### ISSUE THREAD

**Issue by rtroncy** [silknow/crawler#2]
Hey @ehrhart 

It seems that a number of fields / values are systematically visible on the museum web sites for some objects but are not present in the JSON files the crawler is generating. Nevertheless, those missing information have still be considered in the development of the mappings. This may be due to the web scraping process which is by essence very fragile. All those missing fields / values are **rows in green** in the mapping files available in this [folder](https://drive.google.com/drive/folders/1CkYfI3Vf12z5a0Gf46und_EV02odAees?usp=drive_open).

A concrete example is the row `Département | Mode et textile` (in green) of the [record 1 in the MAD mapping](https://docs.google.com/document/d/1zpNE6iBl33PoqLD-xKutDzBjFxPBJvCWx5wZ-K-5b7o/edit#heading=h.u205gzytsbcv).

Tasks:
- make an inventory of all fields / values not collected by the crawler for each mapping file
- fix the crawler and explain

**Comment by rtroncy** [comment]
See also [this document](https://docs.google.com/document/d/1skL3lgVfAnfiWmSMzjQRi6GNuwZG0ws636h_ntOFFZs/edit?usp=sharing) from @mpuren

**Comment by ehrhart** [comment]
Missing fields have been added.

* imatex: Bibliography, Exhibitions, Other items
* met-museum: Department, Object Type / Material, Geographic Location, Date / Era, Related objects, Rights on images
* mfa-boston: Credit Line
* risd: Culture, Year, Medium (1), Medium (2) Materials, Techniques, Dimensions, Credit, Object number, Projects and publications, Exhibition history, Related, Use

The [list of missing fields per source dataset](https://docs.google.com/document/d/1skL3lgVfAnfiWmSMzjQRi6GNuwZG0ws636h_ntOFFZs/edit) has also been updated. Some fields were already being collected so I've added some comments.
